save-plugins/fix-and-warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fixandwarn/FixAndWarnPlugin.kt

```diff
@@ -85,11 +85,12 @@ class FixAndWarnPlugin(
         // TODO: then warn plugin should look at the fix plugin output for actual warnings, and not execute command one more time.
         // TODO: However it's required changes in warn plugin logic (it's should be able to compare expected and actual warnings from different places),
         // TODO: this probably could be obtained after https://github.com/cqfn/save/issues/164,
-        val warnTestResults = warnPlugin.handleFiles(expectedFiles.map { listOf(it) })
-        return fixTestResults + warnTestResults
+        val warnTestResults = warnPlugin.handleFiles(expectedFiles.map { listOf(it) }).toList()
+        return fixTestResults.asSequence() + warnTestResults.asSequence()
```

Comment:
Why do you need `toList` and then `asSequence`? Sequence has an [overloaded plus](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.sequences/plus.html) operation too

Reply:
I need to force FixPlugin to work before warnings will be restored into expected file, so I apply `toList`, 
What about warnPlugin - it should work fine without `toList` and was just reinsurance, removed

Reply:
Hmm, but if we simply concatenate two sequences, they will be calculated in the same order, right? Is this a reinsurance for the future, when we will do this in parallel?

Reply:
This changes corresponds not really to the parallel.\
I noticed, that without applying `toList` to fix results, fix plugin will executes only in the end of work progress. (when warnings are already restored)\
However, we need him to execute exactly after when warnings will be removed from expected file and before when they will be restored

Reply:
I see, yeah, I don't see a better solution yet. Please add comments about it, it's not obvious